requirements.txt

```diff
@@ -84,7 +84,7 @@ greenlet==0.4.17
 importlib-metadata==3.1.1
 Jinja2==2.11.2
 jmespath==0.10.0
-kombu==3.0.37
+kombu==5.0.2
```

Comment:
Requirement is here https://docs.celeryproject.org/en/v5.0.5/whatsnew-5.0.html#kombu